server/src/test/java/org/elasticsearch/search/rescore/QueryRescorerBuilderTests.java

```diff
@@ -220,8 +221,8 @@ public void testUnknownFieldsExpection() throws IOException {
                 "}\n";
         {
             XContentParser parser = createParser(rescoreElement);
-            Exception e = expectThrows(ParsingException.class, () -> RescorerBuilder.parseFromXContent(parser));
-            assertEquals("Unknown RescorerBuilder [bad_rescorer_name]", e.getMessage());
+            Exception e = expectThrows(XContentParseException.class, () -> RescorerBuilder.parseFromXContent(parser));
```

Comment:
All of these should be `NamedObjectNotFoundException` I think.

Reply:
Yeah, I changed all the occurences, thanks